core/contracts/financial-templates/expiring-multiparty/ExpiringMultiPartyCreator.sol

```diff
@@ -36,69 +36,26 @@ contract ExpiringMultiPartyCreator is ContractCreator, Testable, Lockable {
-    // ensure that the ownership of this collateralTokenWhitelist has been renounced (so it is effectively frozen). One
-    // could also set the owner to the address of the Governor contract, but voters may find that option less preferable
-    // since it would force them to take a more active role in managing this financial contract template.
-    AddressWhitelist public collateralTokenWhitelist;
     // - Address of TokenFactory to pass into newly constructed ExpiringMultiParty contracts
     address public tokenFactoryAddress;
-    // - Discretize expirations such that they must expire on the first of each month.
-    mapping(uint256 => bool) public validExpirationTimestamps;
-    // - Time for pending withdrawal to be disputed: 120 minutes. Lower liveness increases sponsor usability.
-    // However, this parameter is a reflection of how long we expect it to take for liquidators to identify
-    // that a sponsor is undercollateralized and acquire the tokens needed to liquidate them. This is also a
-    // reflection of how long a malicious sponsor would need to maintain a lower-price manipulation to get
-    // their withdrawal processed maliciously (if set too low, it’s quite easy for malicious sponsors to
-    // request a withdrawal and spend gas to prevent other transactions from processing until the withdrawal
-    //  gets approved). Ultimately, liveness is a friction to be minimized, but not critical to system function.
-    uint256 public constant STRICT_WITHDRAWAL_LIVENESS = 7200;
-    // - Time for liquidation to be disputed: 120 minutes. Similar reasoning to withdrawal liveness.
-    // Lower liveness is more usable for liquidators. However, the parameter is a reflection of how
-    // long we expect it to take disputers to notice bad liquidations. Malicious liquidators would
-    // also need to attack the base chain for this long to prevent dispute transactions from processing.
-    uint256 public constant STRICT_LIQUIDATION_LIVENESS = 7200;
 
     event CreatedExpiringMultiParty(address indexed expiringMultiPartyAddress, address indexed deployerAddress);
 
     /**
      * @notice Constructs the ExpiringMultiPartyCreator contract.
      * @param _finderAddress UMA protocol Finder used to discover other protocol contracts.
-     * @param _collateralTokenWhitelist UMA protocol contract to track whitelisted collateral.
      * @param _tokenFactoryAddress ERC20 token factory used to deploy synthetic token instances.
      * @param _timerAddress Contract that stores the current time in a testing environment.
      */
     constructor(
         address _finderAddress,
-        address _collateralTokenWhitelist,
         address _tokenFactoryAddress,
         address _timerAddress
     ) public ContractCreator(_finderAddress) Testable(_timerAddress) nonReentrant() {
-        collateralTokenWhitelist = AddressWhitelist(_collateralTokenWhitelist);
```

Comment:
Do we still need to import AddressWhitelist now?

Reply:
Ah wow I totally misread the fact that `collateralTokenWhitelist` is just an instance of `AddressWhitelist`. Please ignore all my comments about this.